### DISCUSSION TITLE
Linux ARM version not being built for Python

### DISCUSSION THREAD

**Discussion by BoHakio** [sfu-db/connector-x#506] [Q&A]
Hi,

I have the issue that I can install Connectorx on my local Mac ARM laptop, but when I want to run it inside a docker container it won't establish a version higher than 0.2.3.

I looked into it and the Linux ARM version isn't being released. My docker image typically follows a `python:3.10-slim` which uses a Debian. 

In the release [file](https://github.com/sfu-db/connector-x/blob/main/.github/workflows/release.yml#L68), the aarch64 version for Linux has been commented out and the only ARM version being built is the Apple-darwin versions.

Is there anything I can do to help out getting that to run?

**Comment by wangxiaoying** [comment]
Resolved by #781 (support from version 0.4.3a1).
